src/main/scala/is/hail/asm4s/StagedBitSet.scala

```diff
@@ -0,0 +1,37 @@
+package is.hail.asm4s
+
+import org.objectweb.asm.tree._
+import scala.collection.generic.Growable
+
+class StagedBitSet(fb: FunctionBuilder[_]) {
+  private var used = 0
+  private var bits: LocalRef[Long] = null
+  private var count = 0
+
+  def newBit(): SettableBit = {
+    if (used >= 64 || bits == null) {
+      bits = fb.newLocal[Long]("bitset"+count)
```

Comment:
You're using a `newLocal` that doesn't exist in this version.  Remove the string here and put it back in the PR that puts the `newLocal` with a base name in?